### DISCUSSION TITLE
Dell G15 5515

### DISCUSSION THREAD

**Discussion by BrockettRocket** [T-Troll/alienfx-tools#466] [Q&A]
Okay I am desperate at this point. I had oem monochrome orange and the backlighting went out. So I looked up on dell's website and factory you could have 4 zone rgb. So I ordered a replacement, did a bunch of research and found out that you also need the keyboard controller so I bought that as well. Installed everything and well... No rgb. No backlight even. Do you have any suggestions? Is this BIOS related? EC related? Is there anything that I can do to solve this? I guarantee there it is and it's probably Dell BS but I have tried so many different things and no lights whatsoever. Please message me on here Troll if you have specific ideas or something. I assume it's cuz my service tag so I am not getting the required drivers for rgb.

**Comment by T-Troll** [comment]
> I had oem monochrome orange

This one can't be controlled. You need to change both board and keyboard.

**Comment by BrockettRocket** [comment]
I did change the keyboard controller board and keyboard.

**Reply by T-Troll** [reply]
> No rgb. No backlight even

Seems like you don't connect it correctly, you should see rainbow or cyan at system start, but have no power at RGB.

**Reply by BrockettRocket** [reply]
Let me know based on this picture if you see anything off. My previous backlighting went out… maybe my motherboard shorted or something? I’m 99% sure I connected it correctly tho.
![IMG_7713E667-25EC-4328-8FF4-E088DB797920](https://github.com/user-attachments/assets/e693eeb4-fb9f-44e6-9ad4-fad823f4970e)

**Reply by gia955** [reply]
Hi i have same problem on my g15 5530, ordered RGB CONTROLLER BOARD and RGB 4 ZONE BOARD, but nothing to do, just 1 time OPEN RGB detected DELL G15 LED CONTROLLER...after that nothing, but light never worked....ho wto solve this? is something BIOS RELATED? Please someone HELP to make RGB work.

**Reply by T-Troll** [reply]
@gia955 i'm sorry, but i don't know how OpenRGB handle devices.
Do you have "Vendor-defined device" under HID USB devices with VID_187C and PID_0550 or 0551?

@itzToxicNRG 
Looks strange - you have "lighting" connector at MB, but it doesn't assemble. What is it? Tron light?

**Comment by gia955** [comment]
I bought original dell g15 5530 4zone rgb kb...i dont find that vendor code
uner devmgtm, do you know if its compatibile ?

I bought also the mcu board to control rgb, all fits ma i cant detect it



Il giorno sab 1 mar 2025 alle 09:03 T-Troll ***@***.***> ha
scritto:

**Reply by T-Troll** [reply]
VID/PID i share is the common Alienware light chip you must have into the system to control lights.
But you don't...
As you can see, it's USB HID, so it should present into device list then connected. No uncommon driver needed.
In case it not - it just don't work. Cables, MB, etc - something wrong on hardware level.

**Comment by gia955** [comment]
I have  this board + flex to Connect mcu to mb TO mb mcu ingress


Il giorno sab 1 mar 2025 alle 11:46 T-Troll ***@***.***> ha
scritto:

**Reply by T-Troll** [reply]
But it still just not works.
Try to reset BIOS (or maybe even update it - i not sure if it's the same for RGB and non-RGB).
Check connector one more time.
It's hard to guess more if device not detected...

**Comment by gia955** [comment]
I see that bios already updated...i updated twice to be sure...i see that
model with rgb has same bios version, that boar were detecetd only one
time..after never, but never powered on the led

Il giorno sab 1 mar 2025 alle 11:54 T-Troll ***@***.***> ha
scritto:
